src/System.ServiceProcess.ServiceController/src/Configurations.props

```diff
@@ -2,7 +2,7 @@
 <Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <PropertyGroup>
     <BuildConfigurations>
-      netstandard-Windows_NT;
+      netcoreapp-Windows_NT;
```

Comment:
This means UWP no longer gets a valid impl.  I guess that's OK.  I doubt any of the API would work from appcontainer.